docs/sources/mimir/operators-guide/run-production-environment/planning-capacity.md

```diff
@@ -154,6 +154,20 @@ For more information about disk requirements, refer to [Compactor disk utilizati
 sum(prometheus_tsdb_head_series)
 ```
 
+There are two compactor specific settings related to working with large tenants that should be considered when planning/adjusting capacity.
+
+These are:
+`-compactor.split-groups`
+`-compactor.split-and-merge-shards`
+
+For more information on how they work, refer to [Compaction algorithm]({{< relref "../architecture/components/compactor/index.md#compaction-algorithm" >}}).
+
+We currently recommend 1 shard per every 8 million active series in a tenant.
+
+For example, for a tenant with 100 million active series, use approximately 12 shards. The shard count should be an even number.
+
+We also recommend setting the number of split-groups to be the same as the shard count.
```

Comment:
Split-groups is a function of number of ingesters that single tenant uses. As such it depends on replication factor. Ideally we would be splitting such that each group has not more than ~20 blocks.

I think formula should be `groups = (series × replication_factor / series_per_ingester) / blocks_per_group`.

For 100M series, 3x replication and 1.5M series per ingesters, if we want to target 20 blocks per group that would be 10 groups. (With 12 groups we get ~17 blocks per group, that's not too bad)